backend/src/main/java/com/festago/fcm/application/FCMNotificationEventListener.java

```diff
@@ -83,7 +83,9 @@ private void checkAllSuccess(BatchResponse batchResponse, Long memberId) {
         List<SendResponse> failSend = batchResponse.getResponses().stream()
             .filter(sendResponse -> !sendResponse.isSuccessful())
             .toList();
-
+        if (failSend.isEmpty()) {
+            return;
+        }
```

Comment:
```suggestion
        if (batchResponse.getFailureCount() > 0) {
            List<SendResponse> failSend = batchResponse.getResponses().stream()
                .filter(sendResponse -> !sendResponse.isSuccessful())
                .toList();
            log.warn("member {} 에 대한 다음 요청들이 실패했습니다. {}", memberId, failSend);
        }
```
다음과 같이 최적화 할 수 있을 것 같아요!